src/reverse/WeakReference.cpp

```diff
@@ -1,4 +1,4 @@
-#include <stdafx.h>
+#include "stdafx.h"
```

Comment:
why did you change all of these? please, revert these. it may make some sense if it is in src folder, but definitely not in reverse sub folder. relative path is ../stdafx.h then.
i used <> everywhere for consistency sake tbh, doesnt have to be like that in every file... but this is misleading someone reading the code.

Reply:
and it just makes this a bit hard to review tbh, just revert these please.